dockerfile/build_environment.Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM gradle:7.1.0-jdk11 AS image
```

Comment:
What will happen if you change the version of Gradle in your project?

Reply:
There is no need to pull gradle from image as it is already in project.
Fixed